docfx-util.ps1

```diff
@@ -0,0 +1,57 @@
+
```

Comment:
I know nothing about PowerShell scripts, so I might be doing something wrong, but if I try to naively run the commands listed above (specifically `./docfx-util.ps1 -c`) I get the following error:

![image](https://github.com/user-attachments/assets/739c7392-b6f3-46c6-9432-7a033bba89fa)

Reply:
I don't know what's going on here

Reply:
brandon let me know that this was user error, so I will mark resolved

Reply:
User error. This popped up when trying to naively run from Git Bash, but worked correctly when called from a PowerShell terminal.

Reply:
though it requires a note in the readme I think, that all commands must be run in powershell